arteria/models/runfolder_utils.py

```diff
@@ -34,14 +35,19 @@ def list_runfolders(monitored_directories, filter_key=lambda r: True):
 
```

Comment:
I don't think the if statement is necessary in the `except-`block above, i.e. if the `if-`statement is false, is will also continue to the next iteration of the loop.

If this is the behavior you want, the code should probably be:
```python
            except AssertionError as e:
                continue
```

If you want to forward the exception if it contains a different message, the code should probably look something like below, and you should add a unit test to check the exception is forwarded correctly
```python
            except AssertionError as e:
                if str(e) != f"File [Rr]unParameters.xml not found in runfolder {subdir}":
                    raise
```